scripts/zkai_d64_external_adapter_surface_probe.py

```
@@ -0,0 +1,634 @@
+        "dependency_probe",
+        "dependency_probe_commitment",
+        "exact_semantic_requirements",
+        "exact_semantic_requirements_commitment",
+        "float_drift_summary",
+        "candidate_adapters",
+        "candidate_matrix_commitment",
+        "conclusion",
+        "non_claims",
+    ):
+        if field not in payload:
+            raise D64ExternalAdapterProbeError(f"missing payload field: {field}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)